src/client/components/UnreadMessages/CountUnreadMessages.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import './Unread.style.css';
+import { useParams } from 'react-router-dom';
+
+function CountUnreadMessages() {
+  const { userId, channelId } = useParams();
+
+  const [unreadMessages, setUnreadMessages] = useState([]);
+
+  useEffect(() => {
+    let mounted = true;
+    if (mounted) {
+      const unReadedMessages = async () => {
+        try {
+          const getUnreadMessages = await fetch(
+            `/api/unread?userId=${userId}&channelId=${channelId}`,
+          ).then((response) => response.json());
+
+          setUnreadMessages(getUnreadMessages);
+        } catch (erro) {
+          return erro;
+        }
+      };
+
+      unReadedMessages();
+    }
+
+    return () => {
+      mounted = false;
+    };
+  }, [userId, channelId]);
+
+  const courentUnreadMessages = [];
+  const unreads = unreadMessages
+    .map((unreadM) => {
```

Comment:
`unreadM` is not descriptive enough. If you cannot use a good name, feel free to just use `u` for example.